mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping

```diff
@@ -45,7 +45,7 @@ CLASS net/minecraft/unmapped/C_wpfizwve net/minecraft/entity/effect/StatusEffect
 		ARG 7 hiddenEffect
 	METHOD <init> (Lnet/minecraft/unmapped/C_wpfizwve;)V
 		ARG 1 that
-	METHOD compareTo (Ljava/lang/Object;)I
+	METHOD compareTo compareTo (Ljava/lang/Object;)I
```

Comment:
Is there a reason "compareTo" is here twice?

Reply:
The obfuscated name is compareTo, I just made it a proper mapped name